backend/app/controllers/api/v1/admin/wage_chunks_controller.rb

```diff
@@ -3,22 +3,28 @@
 class Api::V1::Admin::WageChunksController < ApplicationController
     # POST /wage_chunks
     def create
-        wage_chunk_id =
-            WageChunk.upsert!(wage_chunks_create_params, returning: %i[id])
-        render_succcess WageChunk.find(wage_chunk_id)
+        upsert
```

Comment:
The positions and sessions positions controller do the same thing but can we just get rid of the `create` function here if all it does is call the `upsert` function?

Reply:
That is a good point. However, I am under the impression that `create` function is the more standard name; I have moved the `upsert` function body inside of `create` function, so that `create` function still do the same thing without calling another function